src/main/java/frc/robot/subsystems/LiftSubsystem.java

```diff
@@ -13,29 +19,39 @@
 import edu.wpi.first.wpilibj.command.InstantCommand;
 import edu.wpi.first.wpilibj.command.Subsystem;
 import edu.wpi.first.wpilibj.interfaces.Potentiometer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import frc.robot.Robot;
 import frc.robot.RobotMap;
 
 public class LiftSubsystem extends Subsystem {
-    private SendableSpeedController motor;
+    private CANSparkMax motor;
     private DoubleSolenoid brake;
-    private SendableSpeedController armMotor;
-    private Encoder heightEncoder;
+    private SparkMaxCounter heightEncoder;
     private DigitalInput lowerLimit;
     private DigitalInput upperLimit;
-    private Potentiometer manipAngle;
+    // private CANSparkMax motor;
```

Comment:
Delete this line if you have the same thing up above

Reply:
🐈